EventKit/eventkit-cloud
# Improve clear queues

Clearing queues would request all queues, and if there were enough queues, some rabbitmq api urls might timeout.  This allows for paginating the results.   Likewise there would be issues using the celery app to delete the queues, so this branch just updates the delete calls to use the API url instead.  Lastly this PR adds a management command to make it straightforward to call these commands manually for the sake of maintenance. 

To test run the management command
```
python manage.py clear_rabbitmq
```
and ensure that the empty queues are deleted also try it with --force or --all.
